### ISSUE TITLE
Nullable preferences

### ISSUE THREAD

**Issue by skatset** [chibatching/Kotpref#11]
I use your library and it seems a proper way to work with preferences from Kotlin.

However I found one disadvantage: it is not possible to put the null value to a preference (see example bellow). In origin Android preference model I can do it through the Editor interface. In this case, when I read this property, I retrieve null or default value (if specified).

> public object DevicePreference : KotprefModel() {
    var key: String? by stringPrefVar(default = "")
}

In current case I should to write uncomfortable setting code:

> string: String? = ""
DevicePreference.key = string ?: ""

I think that you should allow to set the null value to a preference. In this case it less comfortable to read value (need to add "!!" symbols) but users should chose preferable way by yourself.

**Comment by chibatching** [comment]
@skatset Hi, I implemented nullable string support. Is this good for your use case?
```kotlin
object UserInfo : KotprefModel() {
    var name: String by stringPrefVar()
    var code: String? by stringNullablePrefVar()
```

**Comment by skatset** [comment]
Thank you! It is what I need

**Comment by chibatching** [comment]
Thanks your feedback. Please wait next release.

**Comment by chibatching** [comment]
Released with 1.1.0
